spec/requests/api/v1/moves_controller_index_spec.rb

```diff
@@ -91,6 +91,25 @@
         # rubocop:enable RSpec/ExampleLength
       end
 
+      context 'with a booked move' do
```

Comment:
AFAIK contexts are used to describe specific contexts that affect the behaviour of the code. My sense is that this test and context doesn't add any value as far as testing behaviour is concerned? I could be wrong, though.

Reply:
I mean, nothing really varies that could change the behaviour of how we index?

Reply:
I think you're capturing this in the finder, spec, too :).